examples/tools/scene_viewer.rs

```diff
@@ -402,7 +438,7 @@ fn camera_controller(
 
     if let Ok((mut transform, mut options)) = query.get_single_mut() {
```

Comment:
Since you removed the camera_check, does that mean we could use `.single_mut()` here? It looks to me like there will now always be a camera so we don't need to handle the `None` case and it would reduce the nesting a bit.

Reply:
Wait, check_camera_controller is unrelated actually, but is there a situation where there's no camera here?